2019/spring/Sterne-TT@11am/yuhao-chen.md

```diff
@@ -0,0 +1,7 @@
+### Yuhao Chen
+Hi! My name is **Yuhao Chen**, a student from Minerva class of *2021*.
+My favourite animes are:
+-[One Piece](https://en.wikipedia.org/wiki/One_Piece)
++[Naruto](https://en.wikipedia.org/wiki/Naruto)
+-[Detective Conan](https://www.detectiveconanworld.com/wiki/Main_Page)
+Look forward to talking with you!
```

Comment:
Looking* instead of look

Also I should I watch Detective Conan!

Reply:
### That should be "I should watch"
👎